lesson_3/homework5.py

```diff
@@ -0,0 +1,12 @@
+# give copy of dictionary or list to function
+def copy_func(x):
```

Comment:
Research how to make a copy on google. Provide an example of incorrect work of your function.